domain/dagconfig/consensus_defaults.go

```diff
@@ -7,18 +7,49 @@ import (
 
 const (
 	defaultMaxCoinbasePayloadLength                = 150
+	// defaultMaxBlockSize is a bound on the size of a block in bytes, larger values increase the bound d
+	// on the round trip time of a block, which affects the other parameters as described below
 	defaultMaxBlockSize                            = 1_000_000
+	// defaultMaxBlockParents is the number of blocks any block can point to.
+	// Should be about d/defaultTargetTimePerBlock where d is a bound on the round trip time of a block.
 	defaultMaxBlockParents                         = 10
 	defaultMassPerTxByte                           = 1
 	defaultMassPerScriptPubKeyByte                 = 10
 	defaultMassPerSigOp                            = 10000
+	// defaultMergeSetSizeLimit is a bound on the size of the past of a block and the size of the past
+	// of its selected parent. Any block which violates this bound is invalid.
+	// Should be at least an order of magnitude smaller than defaultFinalityDuration/defaultTargetTimePerBlock.
+	// (Higher values make pruning attacks easier by a constant, lower values make merging after a split or a spike
+	// in block take longer)
 	defaultMergeSetSizeLimit                       = 1000
 	defaultMaxMassAcceptedByBlock                  = 10000000
 	defaultBaseSubsidy                             = 50 * constants.SompiPerKaspa
 	defaultCoinbasePayloadScriptPublicKeyMaxLength = 150
+	// defaultGHOSTDAGK is a bound on the number of blue blocks in the anticone of a blue block. Approximates the maximal
+	// width of the network.
+	// Formula (1) in section 4.2 shows how to calculate defaultGHOSTDAGK. The delta term represents a bound on the expected
+	// fraction of the network life in which the width was higher than defaultGHOSTDAGK. The current value of K was calculated
+	// for d = 5 seconds and p = 0.05.
 	defaultGHOSTDAGK                               = 18
+	// defaultDifficultyAdjustmentWindowSize is the number of blocks in a block's past used to calculate its difficulty
+	// target.
 	defaultDifficultyAdjustmentWindowSize          = 2640
+	// defaultTimestampDeviationTolerance is the allowed deviance of an inconming block's timestamp, measured in block delays.
+	// A new block can't hold a timestamp lower than the median timestamp of the (defaultTimestampDeviationTolerance*2-1) blocks
+	// with highest accumulated blue work in its past, such blocks are considered invalid.
+	// A new block can't hold a timestamp higher than the local system time + defaultTimestampDeviationTolerance/defaultTargetTimePerBlock,
+	// such blocks are delayed.
```

Comment:
We currently don't delay blocks. We reject them without marking as invalid instead.